packages/next-rest-framework/generate.ts

```diff
@@ -0,0 +1 @@
+export * from './src/generate';
```

Comment:
This root `generate` file allows us to import with `next-rest-framework/generate` instead of exporting directly from the index file, which prevents `esbuild` from being included in the main bundle. Open to suggestions, but I don't think we want to include `esbuild` in the main bundle.